core-web/libs/sdk/client/src/lib/client/content/methods/get-collection/index.ts

```diff
@@ -0,0 +1,146 @@
+import { QueryBuilder } from '../../../../query-builder/sdk-query-builder';
+import { Equals } from '../../../../query-builder/utils/lucene-syntax/Equals';
+import { CONTENT_API_URL, CONTENT_TYPE_MAIN_FIELDS } from '../../const';
```

Comment:
`CONTENT_API_URL` don't need to be a const when you only use it in once place. Can easily be a class property.